cmd/cluster/create.go

```diff
@@ -73,18 +80,21 @@ func NewCreateCommand() *cobra.Command {
 		Namespace:          "clusters",
 		Name:               "example",
 		ReleaseImage:       releaseImage,
+		BaseDomain:         "",
 		PullSecretFile:     "",
 		AWSCredentialsFile: "",
 		SSHKeyFile:         "",
 		NodePoolReplicas:   2,
 		Render:             false,
 		InfrastructureJSON: "",
 		Region:             "us-east-1",
-		InfraID:            "",
+		InfraID:            fmt.Sprintf("%s-%s", "example", utilrand.String(5)),
```

Comment:
Why was this changed? The defaulting is [here](https://github.com/openshift/hypershift/blob/745a6c3a4b6f098281e56623fbfdc7f89784e171/cmd/cluster/create.go#L204) and setting a default on the flag like this makes the CLI present a lie to the user (as the printed value is random per invocation; running `--help` would show one value, but by actually running it again would use a different random value). What problem was changing this solving?